identity-access-manager-keycloak/package-metadata.json

```diff
@@ -21,11 +21,20 @@
     "KC_POSTGRESQL_PASSWORD": "instant101",
     "KC_POSTGRESQL_USERNAME": "admin",
     "KC_POSTGRESQL_DATABASE": "keycloak",
+    "KC_GRAFANA_SSO_ENABLED": "true",
```

Comment:
I think we should probably have it disabled by default and then have the implementation decide on whether or not the specific integration with KC is required. What do you think?